### ISSUE TITLE
"Failed to download JVM" error while downloading OpenJDK for JRuby

### ISSUE THREAD

**Issue by hromadkom** [cloudfoundry/ruby-buildpack#79]
Hi,
I am having some trouble with deploying JRuby application on IBM Bluemix Cloud Foundry.

What version of Cloud Foundry and CF CLI are you using? (i.e. What is the output of running `cf curl /v2/info && cf version`?
`cf version 6.26.0+9c9a261fd.2017-04-06`

```
{
   "name": "Bluemix",
   "build": "235029",
   "version": 0,
   "description": "IBM Bluemix",
   "api_version": "2.54.0",
  .....
}
```

What version of the buildpack you are using?
I suppose it is latest version. I am using https://github.com/cloudfoundry/ruby-buildpack.git as `buildpack` in `manifest.yml`

If you were attempting to accomplish a task, what was it you were attempting to do?
I want to succesfully deploy a JRuby version of my application.

What did you expect to happen?
I expected successfull deployment of my app as I experienced with pure Ruby version of my app using same buildpack.

What was the actual behavior?
```
...
Creating container
Downloading app package...
Downloaded app package (44.3M)
Downloading build artifacts cache...
Downloaded build artifacts cache (46.5M)
Staging...
-------> Buildpack version 1.6.38
       Downloaded [https://buildpacks.cloudfoundry.org/dependencies/bundler/bundler-1.14.6-d7236b1f.tgz]
-----> Supplying Ruby/Rack
       Downloaded [https://buildpacks.cloudfoundry.org/dependencies/jruby/jruby-9.1.8.0_ruby-2.3.1-linux-x64-e6a45d6e.tgz]
-----> Using Ruby version: ruby-2.3.1-jruby-9.1.8.0
-----> Installing JVM: openjdk-8
 !
 !     Failed to download JVM: openjdk1.8-latest.tar.gz
 !     
 !     If this was a custom version or URL, please check to ensure it is correct.
 !
Failed to compile droplet
Exit status 223
Staging failed: Exited with status 223
Destroying container
Successfully destroyed container

FAILED
BuildpackCompileFailed
...
```

I suppose there might be some problem with access to S3 while downloading the OpenJDK. 

Thanks

**Comment by dgodd** [comment]
Unfortunately the version of openjdk which was published online was changed.

Going forward:
1. For new buildpacks we are copying it to our own bucket to ensure consitency.
2. We are seeing if the team which owns that file can re-upload it

I think if you use the latest version of the buildpack then your problem will be fixed

**Comment by sclevine** [comment]
Fixed in 1.6.39. See also: #80
